golang/seedGenerator/main.go

```diff
@@ -0,0 +1,16 @@
+package main
```

Comment:
similar to the `pkg` folder convention, these folder separated main classes could be shifted underneath a `cmd` folder - might be more idiomatic go